extras/rya.geoindexing/geo.mongo/src/test/java/org/apache/rya/indexing/geotemporal/mongo/MongoITBase.java

```diff
@@ -1,64 +0,0 @@
-/**
```

Comment:
So it looks like you consolidated MongoITBase and MongoTestBase, which is great.  However, since MongoTestBase via EmbeddedMongoSingleton spawns new Mongo processes, it is by definition an Integration Test and should probably be renamed to MongoITBase for clarity.  Also, tests that use it (EntityQueryNodeTest, EventQueryNodeTest, MongoEventStorageTest, etc) should probably be renamed with an IT suffix so they can be exercised and cleaned up by the maven-failsafe-plugin.

Reply:
Renamed it to MongoITBase and updated the names of things that extend it.